src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.getRequestDispatcher("/WEB-INF/views/addDriverToCar.jsp")
+                .forward(request, response);
+
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+        Long carId = Long.valueOf(request.getParameter("car-id"));
+        Long driverId = Long.valueOf(request.getParameter("driver-id"));
```

Comment:
```suggestion
        Long carId = Long.valueOf(request.getParameter("car_id"));
        Long driverId = Long.valueOf(request.getParameter("driver_id"));
```
i think as code convention, better change to this